docs/UserGuide.md

```diff
@@ -450,7 +499,7 @@ Examples:
 
 #### 5.3.3 Delete a product : `delProduct`
```

Comment:
I think you forgot to change the header here

Reply:
yea, not focusing on changing the header here just yet.